hpat/tests/test_basic.py

```diff
@@ -297,6 +297,7 @@ def test_array_reduce(self):
             self.assertEqual(count_array_OneDs(), 0)
             self.assertEqual(count_parfor_OneDs(), 1)
 
+    @unittest.skipIf(numba.__version__.startswith('0.46.'), "Broken in numba 0.46.x")
```

Comment:
add url to issue https://github.com/numba/numba/issues/4690

And IMO better to check exact version, i.e. if it will be still broken on 0.46.1 we will see that

Reply:
Also, maybe create some helper function to check numba version to avoid copypaste